app/controllers/datastreams_controller.rb

```diff
@@ -4,21 +4,15 @@ class DatastreamsController < ApplicationController
   include Blacklight::Catalog
   copy_blacklight_config_from CatalogController
 
-  before_action :show_aspect, only: %i[dc show edit]
-
-  def dc
-    respond_to do |format|
-      format.html { render layout: !request.xhr? }
-    end
-  end
+  before_action :show_aspect, only: %i[show edit]
 
   def edit
     @content = @object_client.metadata.datastream(params[:id])
     render layout: !request.xhr?
   end
 
   def show
-    if params[:dsid] == 'full_dc'
```

Comment:
Can you say more about the reason for this change?

Reply:
This was broken, if you go to "View Full Dublin Core" and then click "View XML" it was trying to hit this path, but missing because some other PR changed the parameter names.